app/cells/folio/console/layout/sidebar_cell.rb

```diff
@@ -19,10 +19,17 @@ def filtered_link_groups_with_links
     link_groups.filter_map do |group|
       I18n.with_locale(group[:locale] || I18n.locale) do
         if group[:links].present?
-          links = group[:links].filter_map { |l| link_from(l) }
+          subgroups = group[:links].slice_before(:hr).to_a
```

Comment:
`slice_before` to rozdeli jen na dve pole, ne? Co kdyz to chci mit rozdelene na vice casti?

Reply:
slice_before rozdeluje input na chunky podle separatoru, takze je to rozdelene na vice casti

Reply:
rozdeleni podle hr jsem delal kvuli zadani issue od Lukase, to poradi a rozdeleni co tam je je proste namichane mišmaš a tohle reseni se mi libi v tom, ze je to customizable a da se to overridnout i v auctify, tvoje reseni mi prijde slozite a nerozumim tomu